lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -122,6 +122,8 @@ def encoded_string(string)
         end
 
         def indent_lines(lines, failure_number)
+          return lines if failure_number.nil?
```

Comment:
The problem is the presenter was controlled elsewhere, and it's only this formatter that wants no leading indentation (it's pointless in html anyway) so I've handled this with the cleanup regexp instead.